tests/testthat/test-formatters.R

```diff
@@ -808,22 +808,22 @@ test_that("All supported 1d format cases of decimal alignment", {
   expected <- c(
     "           mpg           wt                         left   ",
     "-----------------------------------------------------------",
-    "a          11                     11              decimal  ",
-    "b          11.           11.                      left     ",
-    "c          11.1                            11.1   right    ",
+    "a          11                     11               decimal ",
+    "b          11.           11.                        left   ",
+    "c          11.1                            11.1     right  ",
```

Comment:
All these changes in snapshots depend on the change of default for value cells (which is "center" in {rtables}, different from what it was here)